app/app.rb

```diff
@@ -69,7 +69,14 @@ def validate_dates(space)
       end
     end
 
+    def validate_space(space)
+      if current_user.id == space.user_id
+        redirect('/spaces')
+      end
+    end
+
     def validate(space)
+      validate_space(space)
       validate_dates(space)
       booking_from = space.bookings.map(&:from_date)
       booking_to = space.bookings.map(&:end_date)
```

Comment:
just a little thing but is it maybe worth renaming one of this to something a  little bit different?  Validate_space(space) and validate(space) are a little bit hard to follow?